### PR TITLE
Added types

### PR DESCRIPTION
**Pull request by zach-albia**
I'm opening this PR as per [issue #24](https://github.com/sarahdayan/dinero.js/issues/24). I understand that adding the types file to the project may not be to your liking. If so, please advise if you would prefer them to be outside the repo instead, e.g. in DefinitelyTyped.

### CODE REVIEW

**Comment by Bessonov** [comment]
Hey @zach-albia, I miss `types` entry in package.json

File: src/dinero.d.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  hasSameAmount(comparator: Dinero): boolean;
+  toFormat(format?: string, roundingMode?: dinero.RoundingMode): string;
+  toUnit(): number;
+  toRoundedUnit(digits, roundingMode?: dinero.RoundingMode): number;
+  toObject(): dinero.ObjectLiteral;
+  normalizePrecision(objects: Dinero[]): Dinero[];
+}
+
+declare module 'dinero.js' {
+  export = Constructor
```
Comment (BirdInTheCity) [inline src/dinero.d.ts]: I'm getting errors about needing a default export. It worked for me when changed to:
`export default Constructor`

Comment (zach-albia) [reply src/dinero.d.ts]: Hey, thanks for catching that. will update pull request accordingly.

**Comment by zach-albia** [comment]
I'm looking to add the following to package.json line 15 : `"types": "./src/dinero.d.ts",` as per comment by @Bessonov. I don't know what line 14 `"main": "build/cjs/dinero.js",` does but it looks like it does something related to CommonJS.

Should I instead set the `types` attribute to something like `"types": "build/cjs/dinero.d.ts",`?

**Pull request closed without merging**